lib/app.dart

```diff
@@ -87,6 +86,7 @@ class AppState extends State<App> with WidgetsBindingObserver, NavigatorObserver
     WidgetsBinding.instance?.addObserver(this);
     _listenForDataClear();
     _listenScreenshotPref();
+    _handleUpdateSchemes();
```

Comment:
I don't think `initState()` is called when the app comes from background. So that would mean that the app updates its schemes on start-up now, but not on resume anymore.

Reply:
Exactly, that was the idea. It is a more logical moment and also easier to explain to users this way if we would want them to refresh/update the scheme. But we could do both (didn't do that because of the possible load increase on the server at pbdf), let's discuss!

Reply:
Re-introduced _handleUpdateSchemes being called at app resume. Now it checks at both app startup and resume